sdk/servicebus/service-bus/src/serviceBusClient.ts

```diff
@@ -196,6 +201,7 @@ export class ServiceBusClient {
    * @param {ServiceBusClientOptions} options - Options to control ways to interact with the
    * Service Bus Namespace.
    * @returns {ServiceBusClient}
+   * @throws {Error} Thrown if the connection string contents were malformed.
```

Comment:
`createFromTokenProvider` does not take connection string

Reply:
updated